commands.py

```diff
@@ -32,7 +32,7 @@ def inner(func, description=description):
 
 """
 Each @is_command function is a command (!!), callable by sending "!<function_name>" in chat.
-All replies will be sent in the bot's colour, using /me unless specified otherwise.
+All replies will be sent in the bots colour, using /me unless specified otherwise.

```

Comment:
????????? This was correct :O

Reply:
Eh, I just smacked the squiggly blue line from VSC on this one, you right though.